components/Button.tsx

```diff
@@ -3,21 +3,32 @@ import { FC, PropsWithChildren } from 'react';
 interface Props {
   form?: string;
   disabled: boolean;
+  variant: string;
```

Comment:
Since this prop should only be one of two strings `inputForm` or `nextHoverButton`, you could type this like so:
```suggestion
  variant: 'inputForm' | 'nextHoverButton';
```